### PR TITLE
Add Jason PPP

### PR DESCRIPTION
**Pull request by jason-ng-zq99**

### CODE REVIEW

File: docs/team/jason-ng-zq99.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+* **Credits**: The design structure is inspired from AddressBook Level 3
+
+**Enhanced Feature**: Enhanced the ability to find person. (PRs [#51](https://github.com/AY2122S1-CS2103T-T09-2/tp/pull/51), [#76](https://github.com/AY2122S1-CS2103T-T09-2/tp/pull/76), [#84](https://github.com/AY2122S1-CS2103T-T09-2/tp/pull/84))
+* **What it does**: Allows the user to find person from contHACKS with different information quickly by allowing partial word matches.
+* **Justification**: User now need not type the full name just to search for a single person, making the process faster especially for long names.
+* **Justification**: While this might return more entries per search, it is still not significantly long enough to affect search experience.
+
+**Enhanced Feature**: Refactor `moduleClass` class to `moduleLesson` class. (PR [#98](https://github.com/AY2122S1-CS2103T-T09-2/tp/pull/98))
+* **What it does**: Rename `moduleClass` class to `moduleLesson` class. This includes changing all instances of `moduleClass` in pacakages, methods, test cases and comments.
```
Comment (tanruiquan) [inline docs/team/jason-ng-zq99.md:24]:
```suggestion
* **What it does**: Rename `moduleClass` class to `moduleLesson` class. This includes changing all instances of `moduleClass` in packages, methods, test cases and comments.
```

**Review by tanruiquan — APPROVED** [review]
LGTM 👍

File: docs/team/jason-ng-zq99.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+* **Highlights**: On top of searching using the lesson's module code, you can also search via the lesson's start time and the day of the week of the lesson. The once obsolete tag prefix `t/` was recycled to be used to search for lesson start time.
+* **Credits**: The design structure is inspired from AddressBook Level 3
+
+**Enhanced Feature**: Enhanced the ability to find person. (PRs [#51](https://github.com/AY2122S1-CS2103T-T09-2/tp/pull/51), [#76](https://github.com/AY2122S1-CS2103T-T09-2/tp/pull/76), [#84](https://github.com/AY2122S1-CS2103T-T09-2/tp/pull/84))
+* **What it does**: Allows the user to find person from contHACKS with different information quickly by allowing partial word matches.
+* **Justification**: User now need not type the full name just to search for a single person, making the process faster especially for long names.
+* **Justification**: While this might return more entries per search, it is still not significantly long enough to affect search experience.
+
+**Enhanced Feature**: Refactor `moduleClass` class to `moduleLesson` class. (PR [#98](https://github.com/AY2122S1-CS2103T-T09-2/tp/pull/98))
```
Comment (driip-ddrop) [inline docs/team/jason-ng-zq99.md:23]: I think this is under team tasks not feature

File: docs/team/jason-ng-zq99.md (hunk 1 above)
Comment (jason-ng-zq99) [reply docs/team/jason-ng-zq99.md]: Thanks!

File: docs/team/jason-ng-zq99.md (hunk 2 above)
Comment (jason-ng-zq99) [reply docs/team/jason-ng-zq99.md]: Yes! thanks for the catch, resolved in 538d75c

**Pull request merged**